todo_app/app.py

```diff
@@ -1,5 +1,7 @@
+from pprint import pprint
```

Comment:
Unused import.

Also you may find live debugging an easier alternative to printing stuff when you're trying to work out what's going on/what the response looks like.